src/payment/cart/TotalTable.jsx

```diff
@@ -4,14 +4,22 @@ import { FormattedMessage } from '@edx/frontend-platform/i18n';
 
 import LocalizedPrice from './LocalizedPrice';
 
-const TotalTable = ({ total }) => (
+const TotalTable = ({ total, isSubscription }) => (
   <div className="summary-row font-weight-bold d-flex">
     <span className="flex-grow-1">
-      <FormattedMessage
-        id="payment.summary.table.label.total.to.pay"
-        defaultMessage="TOTAL"
-        description="Label for the final total price of an order."
-      />
+      {!isSubscription ? (
```

Comment:
Just a small preference to also follow `SummaryTable` component, swap the formatted messages so the ternary operator is `isSubscription ? ...`

Reply:
Done